src/style.css

```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+}
+/* stylelint-disable max-empty-lines */
+/* stylelint-disable color-hex-length */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable shorthand-property-no-redundant-values */
```

Comment:
Don't disable the eslint, fix the problems. 
Try googling errors' text to understand what they mean